box/5.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,111 @@
+
+== Prerequisites
+
+Before creating an app, you must have access to the Box target resource and Anypoint Platform.
+You must also understand how to create a Mule app using Anypoint Studio.
+
+You should be familiar with the use of the Box 2.0 API, Mule, Anypoint Connectors, Anypoint Studio, Mule concepts, elements in a Mule flow, and global elements.
+
+To use this connector you need the following:
+
+* A developer account or other valid Box instance.
+* MuleSoft Enterprise License and necessary entitlements.
+
+
+== Supported Operations
+
+This is a list of supported operations grouped under a category that the Box connector supports.
+
+[%header%autowidth.spread]
+|===
+|Category |Supported Operations
+
+| Collaborations a| * Create collaboration
+                    * Delete collaboration
+                    * Get collaboration
+                    * Get Pending collaborations
+                    * Update collaboration
+
+| Comments a| * Create comment
+              * Delete comment
+              * Get comment
+              * Update comment
+
+| Files a| * Copy file
+           * Delete file
+           * Get file
+           * Get file comments
+           * Get file content
+           * Get file tasks
+           * Get file thumbnail
+           * Update file info
+
+| File Versions a| * Delete file version
+                   * Get file versions
+                   * Promote file version
+
+| Folders a| * Copy folder
+             * Create folder
+             * Delete folder
+             * Get folder
+             * Get folder collaborations
+             * Get folder items
+             * Update folder
+
+| Groups a| * Create group
+            * Create group membership
+            * Delete group
+            * Delete group membership
+            * Get enterprise groups
+            * Get group
+            * Get group membership
+            * Get group memberships
+            * Get user group membership
+            * Update group membership
+
+| Search a| * Search
+
+| Tasks a| * Create task
+           * Create task assignment
+           * Delete task
+           * Delete task assignment
+           * Get task
+           * Get task assignment
+           * Get task assignments
+           * Update task
+           * Update task assignment
+
+| Trash a| * Delete trashed file
+           * Delete trashed folder
+           * Get trashed file
+           * Get trashed folder
+           * Get trashed items
+           * Restore trashed file
+           * Restore trashed folder
+
+| Users a| * Create user
+           * Delete user
+           * Get enterprise users
+           * Get user
+           * Update user
+           * Update user folder
+
+|===
+
+== Next Step
+After you complete the prerequisites and experiment with templates and examples, you are ready to create an app with xref:box-connector-studio.adoc[Anypoint Studio].
```

Comment:
```suggestion
After you complete the prerequisites, you are ready to create an app with xref:box-connector-studio.adoc[Anypoint Studio].
```

Reply:
There is no mention of the examples or templates sections where the user can experiment. Though, you have the link in the See Also section.